qutebrowser/browser/downloads.py

```diff
@@ -1094,7 +1097,8 @@ def download_delete(self, count=0):
 
     @cmdutils.register(instance='download-model', scope='window', maxsplit=0)
     @cmdutils.argument('count', value=cmdutils.Value.count)
-    def download_open(self, cmdline: str = None, count: int = 0) -> None:
+    @cmdutils.argument('dir', flag='d')
```

Comment:
This line isn't needed as there's nothing to customize from the default behavior (the argument name is automatically used, and its first letter is used as flag).